### PR TITLE
Minify Web Assets

### PR DESCRIPTION
**Pull request by navveenb**
Added web patterns

### CODE REVIEW

**Comment by dubrie** [comment]
assigning to @dubrie for initial review

File: docs/catalog/web/minify-web-assets.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+version: 1.0
+submitted_by: navveenb
+published_date: tbd
+category: web
+tags: 
+ - web-assets
```
Comment (dubrie) [inline docs/catalog/web/minify-web-assets.md:7]: `web-assets` is not a supported tag for patterns. You can see a list of available tags at https://patterns.greensoftware.foundation/guide/suggested-tags/ but I think just adding `web` here would be enough.

**Review by dubrie — COMMENTED** [review]
Please update the tags but the structure and clarity is good for the rest of it.

**Comment by dubrie** [comment]
Adding @markus-ntt-seidl for SME review

File: docs/catalog/web/minify-web-assets.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ - role:web-developer
+ - size:small
+---
+
+# Minify web assets
+
+## Description
+A web page comprises various assets, such as CSS, JavaScript, and third-party libraries like jQuery, Bootstrap etc. Minification removes unnecessary or redundant data without affecting how the resource is processed by the web browser.
```
Comment (markus-ntt-seidl) [inline docs/catalog/web/minify-web-assets.md:15]: I would propose to:
* Add HTML, which can also be minified
* Remove the libraries, as these often come minified, and it's a different logical level as HTML/CSS/JS

File: docs/catalog/web/minify-web-assets.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+## Solution
+Minify web assets to reduce page size and network bandwidth while delivering the content. At scale, even a small reduction in size can have a significant impact.
+
+## SCI Impact
+`SCI = (E * I) + M per R`
+
+For the SCI equation, minifying web assets will impact the following::
+- 'E': Minifying  web assets would reduce the size of the web page and lead to reduction in network bytes during transfer of content. The browser/device would takes less energy to render the content. This should reduce the energy consumption of your web page, and consequently, the E number should decrease.
```
Comment (markus-ntt-seidl) [inline docs/catalog/web/minify-web-assets.md:24]: * Typo: "Minifying  web" (looks to me as there is an extra space).
* "would takes" --> "would take"

**Review by navveenb — COMMENTED** [review]
Done with changes. Added HTML as one of the content. Thanks. Have kept the remaining, as other web assets (CSS, javascript, any libraries etc) might not be minified always or the developer forgets to use a minified version.

File: docs/catalog/web/minify-web-assets.md (hunk 3 above)
Comment (markus-ntt-seidl) [reply docs/catalog/web/minify-web-assets.md]: This has not been resolved.

**Comment by markus-ntt-seidl** [comment]
This is still open and not resolved:

* Typo: "Minifying web" (looks to me as there is an extra space).
* "would takes" --> "would take"

**Comment by markus-ntt-seidl** [comment]
@dubrie Can you change the title of the PR to "Minify Web Assets"
This would then close #70 and #72 .

**Pull request merged**